app/stirr/views/version-view.html

```diff
@@ -0,0 +1,13 @@
+<div class="padding" ng-controller="VersionViewController">
+  <div ng-show="showSpinner" ng-include="'_spinner.html'"></div>
+  <super-navigate location="stirr#view" data-params-id="{{ recipe.id }}" ng-repeat="recipe in recipes">
+    <div class="list">
+      <a class="item item-thumbnail-left" href="#">
+        <img ng-src="{{ recipe.image.url || '/food-placeholder.png' }}">
```

Comment:
Any chance that we can use the image resizing logic here? This messes up the aspect ratio.